clang/test/CodeGen/aix-builtin-mapping.c

```diff
@@ -6,17 +6,21 @@
 // RUN: %clang_cc1 -triple powerpc-ibm-aix -mlong-double-64 -emit-llvm -o - %s | FileCheck -check-prefix=CHECK %s
 // RUN: %clang_cc1 -triple powerpc64-ibm-aix -mlong-double-64 -emit-llvm -o - %s | FileCheck -check-prefix=CHECK %s
 
+long double input = 0.0L;
+
 int main()
 {
   int DummyInt;
   long double DummyLongDouble;
   long double returnValue;
 
   returnValue = __builtin_modfl(1.0L, &DummyLongDouble);
-  returnValue = __builtin_frexpl(0.0L, &DummyInt);
+  returnValue = __builtin_frexpl(input, &DummyInt);
```

Comment:
Once the handling of the second parameter is corrected, the change to the test should become unnecessary.